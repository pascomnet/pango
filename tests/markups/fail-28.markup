<span overline="triple">test</span>
