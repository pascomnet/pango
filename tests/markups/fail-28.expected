ERROR: 'triple' is not a valid value for the 'overline' attribute on <span> tag, line 1; valid values are none/single/double/dashed/dotted/wavy